server/controllers/courses/addCourse.js

```diff
@@ -0,0 +1,22 @@
+const insertCourse = require('../../database/queries/addCourse');
+
+exports.addCourse = (req, res) => {
+  insertCourse({ ...req.body })
+    .then((course) => {
+      if (!course.rowCount) {
+        res.send({
```

Comment:
@AbdallahAmmar96 It might be a good idea to set the status to indicate an error occurred (400 or 500)

Reply:
I set the status to 400, not 500 
because the error not from the server, I take it as a bad request
What do you think ??
@tomduggan85

Reply:
@AbdallahAmmar96 yeah, if bad user input is the only reason that insertCourse can fail, then 400 is perfect!